ballerina/tests/department_client.bal

```diff
@@ -30,47 +30,51 @@ client class DepartmentClient {
 
     private SQLClient persistClient;
 
-    public function init() returns error? {
-        mysql:Client dbClient = check new (host = host, user = user, password = password, database = database, port = port);
+    public function init() returns Error? {
+        mysql:Client|sql:Error dbClient = new (host = host, user = user, password = password, database = database, port = port);
```

Comment:
Needs to change to cast the error to 'persist:Error' in case the MySQL client creation fails